lib/board.rb

```diff
@@ -0,0 +1,46 @@
+class Board
+  attr_accessor :board, :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells.clear
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    @cells[input.to_i - 1]
+  end
```

Comment:
Get in the habit of accessing instance and class variables using getter and setter methods when you have them.  In this case, use `cells` instead of `@cells`.